ote/src/cljs/ote/ui/table.cljs

```diff
@@ -48,11 +47,14 @@
                 (when class
                   {:class class}))
      [ui/table-header {:key (str table-id "-hdr")
-                       :style {:overflow "visible" :background-color (color :grey300)}
+                       :style {:overflow "visible"
+                               :background-color (color :grey300)
+                               :border-bottom (str "1px solid " colors/gray650 " !important")}
                        :adjust-for-checkbox false
                        :display-select-all false}
       [ui/table-row {:key (str table-id "-hdr-row")         ;; When adding more rows remember to add their id to key
-                     :style {:overflow "visible"}
+                     :style {:overflow "visible"
+                             :border-bottom (str "1px solid " colors/gray650 " !important")}
```

Comment:
Why only -bottom? Should it be all borders?

Reply:
Border is set for multiple elements but still I think that we wouldn't win anything by adding it to def and merging styles here. 
This border issue comes from material ui. And material-ui seems to add border to table-row item or table-row AND table-header item. So I had to overwrite them in both elements.

Reply:
Yes, the need for setting the style for all these components is not contested here.

Point was duplicate code. 
Duplication leads to someone potentially changing only some/one of the locations in future and missing some. That That could be avoided by 
1) Using a def'd style makes it explicit and developer-proof
2) Using css `inherit` also removes the duplication because in this case child component is meant to follow parent component style without differing from it. 
E.g. On browser dev console setting the `tr` border to "inherit" fixes the row and all it's td's without the need to change the td styles at all.

There wouldn't be any extra effort or complexity on above alternativecs, on the contrary they would be more future-proof and simple.
But okay I'll let this go :)